Signing Scenarios.txt

```diff
@@ -0,0 +1,55 @@
+Signing Scenarios 
```

Comment:
Did you mean to put this in the root of the repo?

Reply:
Thanks, I didn't mean to check that in,.